resources/js/sfx.js

```diff
@@ -0,0 +1,214 @@
+import Prando from "prando";
+import {fy_shuffle, int16_as_bytes} from "./utils";
+
+export default class SFX {
+  constructor(seed) {
+    this.rand = new Prando(seed);
+  }
+
+  create_sfx(name, sfx_set, orig_id, addr, chain, accomp = false) {
```

Comment:
non-blocking: would be nice to have a doc block explaining what all these parameters are